src/vm/analysis/tests/costs.rs

```diff
@@ -90,6 +91,21 @@ pub fn test_tracked_costs(prog: &str) -> ExecutionCost {
             &TEST_HEADER_DB,
             &TEST_BURN_STATE_DB,
         );
+
+        if epoch == StacksEpochId::Epoch2_05 {
+            conn.initialize_epoch_2_05().unwrap();
+        }
+
+        conn.commit_block();
+    }
+
+    {
+        let mut conn = clarity_instance.begin_block(
+            &StacksBlockId([1 as u8; 32]),
+            &StacksBlockId([2 as u8; 32]),
+            &TEST_HEADER_DB,
+            &TEST_BURN_STATE_DB,
+        );
```

Comment:
Do you want to add an assertion here to verify that this block is indeed executing in the proper epoch?

Reply:
Added in https://github.com/blockstack/stacks-blockchain/pull/2924/commits/4972224e49852c4739ac56dd1a5a0c4f34f4221a